documentation/asciidoc/accessories/camera/libcamera_options_common.adoc

```diff
@@ -0,0 +1,388 @@
+
+----
+	--roi,				Select a crop (region of interest) from the camera <x,y,w,h>
+----
+
+The `--roi` (region of interest) option allows the user to select a particular crop from the full field of view provided by the sensor. The coordinates are specified as a proportion of the available field of view, so that `--roi 0,0,1,1` would have no effect at all.
+
+The `--roi` parameter implements what is commonly referred to as "digital zoom".
+
+Example `libcamera-hello --roi 0.25,0.25,0.5,0.5` will select exactly a quarter of the total number of pixels cropped from the centre of the image.
+
+==== Camera Control
+
+The following options affect the image processing and control algorithms that affect the camera image quality.
+
+----
+	--sharpness,			Set image sharpness <number>
+----
+
+The given `<number>` adjusts the image sharpness. The value zero means that no sharpening is applied, the value 1.0 uses the default amount of sharpening, and values greater than 1.0 use extra sharpening.
+
+Example: `libcamera-still -o test.jpg --sharpness 2.0`
+
+----
+	--contrast,			Set image constrat <number>
+----
+
+The given `<number>` adjusts the image contrast. The value zero produces minimum contrast, the value 1.0 uses the default amount of contrast, and values greater than 1.0 apply extra contrast.
+
+Example: `libcamera-still -o test.jpg --contrast 1.5`
+
+----
+	--brightness,			Set image brightness <number>
+----
+
+The given `<number>` adjusts the image brightness. The value -1.0 produces an (almost) black image, the value 1.0 produces an almost entirely white image and the value 0.0 produces standard image brightness.
+
+Note that the brightness parameter adds (or subtracts) an offset from all pixels in the output image. The `--ev` options is often more appropriate.
+
+Example: `libcamera-still -o test.jpg --brightness 0.2`
+
+----
+	--saturation,			Set image colour saturation <number>
+----
+
+The given `<number>` adjusts the colour saturation. The value zero produces a greyscale image, the value 1.0 uses the default amount of sautration, and values greater than 1.0 apply extra colour saturation.
+
+Example: `libcamera-still -o test.jpg --saturation 0.8`
+
+----
+	--ev,				Set EV compensation <number>
+----
+
+Sets the EV compensation of the image in units of _stops_, in the range -10 to 10. Default is 0. It works by raising or lowering the target values the AEC/AGC algorithm is attempting to match.
+
+Example: `libcamera-still -o test.jpg --ev 0.3`
+
+----
+	--shutter,			Set the exposure time in microseconds <number>
+----
+
+The shutter time is fixed to the given value. The gain will still be allowed to vary (unless that is also fixed).
+
+Note that this shutter time may not be achieved if the camera is running at a frame rate that is too fast to allow it. In this case the `--framerate` option may be used to lower the frame rate. The maximum possible shutter times for the official Raspberry Pi supported cameras are:
```

Comment:
Does each camera also have a minimum shutter time? :shrug:  (or is that irrelevant, as it would produce images that are too dark to be useful?)